src/screenshots/covidcast/map_preview.py

```diff
@@ -0,0 +1,234 @@
+      return len(spinners) > 0
+
+    print('waiting for map to load')
+    WebDriverWait(driver, 10).until_not(is_loading)
+
+    root = driver.find_elements_by_css_selector('main.root')[0]
+
+    print('hiding extra ui')
+    options = root.find_elements_by_class_name('options-container')[0]
+    search = root.find_elements_by_class_name('search-container')[0]
+    compare = root.find_elements_by_class_name('panel-bottom-wrapper')[0]
+    driver.execute_script('arguments[0].style.display="none"', options)
+    driver.execute_script('arguments[0].style.display="none"', search)
+    driver.execute_script('arguments[0].style.display="none"', compare)
+
+    print('resetting map view')
+    controls = root.find_elements_by_class_name('map-controls-container')[0]
+    home_button = controls.find_elements_by_class_name('pg-button')[2]
+    home_button.click()
+    # allow time for the animation to complete
+    time_impl.sleep(5)
+
+    hover_location = (args.hover_x, args.hover_y)
+    print(f'hovering mouse at {hover_location}')
+    webdriver_impl.ActionChains(driver).move_to_element_with_offset(
+        root, *hover_location).perform()
+    # allow time for the county info popup to appear
+    time_impl.sleep(1)
+
+    print(f'rendering screenshot at {args.path}')
+    root.screenshot(args.path)
+
+  def get_argument_parser():
+    """Define command line arguments and usage.
+
+    Returns
+    -------
+    argparse.ArgumentParser
+        A parser for command line arguments.
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--path',
+        type=str,
+        required=True,
+        help='absolute path of generated screenshot (e.g. /path/name.png)')
+    parser.add_argument(
+        '--source',
+        type=str,
+        default='doctor-visits',
+        help='COVIDcast data source (default \'doctor-visits\')')
+    parser.add_argument(
+        '--signal',
+        type=str,
+        default='smoothed_adj_cli',
+        help='COVIDcast signal name (default \'smoothed_adj_cli\')')
+    parser.add_argument(
+        '--geo_type',
+        type=str,
+        default='county',
+        help='COVIDcast geographic resolution (default \'county\')')
+    parser.add_argument(
+        '--geo_value',
+        type=str,
+        default='42003',
+        help='COVIDcast location name (default \'42003\')')
+    parser.add_argument(
+        '--date',
+        type=str,
+        help=(
+          'date of the signal in YYYYMMDD format; if unspecified, use the '
+          'most recent date for the signal'
+        ))
+    parser.add_argument(
+        '--width',
+        type=int,
+        default=1800,
+        help='width of the screenshot, in pixels (default 1800)')
+    parser.add_argument(
+        '--height',
+        type=int,
+        default=900,
+        help='height of the screenshot, in pixels (default 900)')
+    parser.add_argument(
+        '--hover_x',
+        type=int,
+        default=0,
```

Comment:
can you set the default values to something useful?

Reply:
updated